packages/app/src/renderer/windows/settings/app.tsx

```diff
@@ -141,10 +168,10 @@ const App = (): JSX.Element => {
               />
             </Form.Item>
             <Form.Item noStyle>
-              <Button onClick={window.electron.ipcRenderer.locateLogFile}>Select</Button>
+              <Button onClick={handleChangeOfPath}>Select</Button>
```

Comment:
@JohannesMerkt  I have no idea why we have to do this ... it was throwing some react errors ...

Reply:
Me neither but good you found out that it causes errors